backend/module/proxy/proxy.go

```diff
@@ -89,6 +92,16 @@ func (m *mod) RequestProxy(ctx context.Context, req *proxyv1.RequestProxyRequest
 		headers.Add(k, v)
 	}
 
+	var requestBody io.ReadCloser
+	if req.Request != nil {
+		requestJSON, err := protojson.Marshal(req.Request)
+		if err != nil {
+			return nil, err
+		}
+		buff := bytes.NewBuffer(requestJSON)
+		requestBody = ioutil.NopCloser(buff)
+	}
```

Comment:
move this below request construction and then you can directly set request.Body instead of maintaining the local var